analyzers/tests/SonarAnalyzer.Test/TestCases/CatchRethrow.vb

```diff
@@ -58,7 +58,7 @@ Namespace Tests.TestCases
 
             Try
                 doSomething()
-            Catch exc As ArgumentException When True 'Noncompliant
+            Catch exc As ArgumentException When True
```

Comment:
```suggestion
            Catch exc As ArgumentException When True  ' FN
```

Reply:
I'll consider the `True` just a scaffolding noise. It's not a SE rule to determine if the expression is always true